pytest_html/plugin.py

```diff
@@ -197,6 +197,8 @@ def append_extra_html(self, extra, extra_index, test_index):
                                              extra.get('extension'))
 
             elif extra.get('format') == extras.FORMAT_TEXT:
+                if isinstance(extra.get('content'), bytes):
+                    extra['content'] = extra['content'].decode("utf-8")
```

Comment:
Can't we do this after assigning `content`? Also, could you include a comment to explain why this is necessary?